mobile/src/bundles/common/components/input/styles.ts

```diff
@@ -3,7 +3,40 @@ import { StyleSheet } from 'react-native';
 const styles = StyleSheet.create({
```

Comment:
Review styles with global styles

Reply:
When we use global styles, it should look something like this? 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/104503014/b4edddab-4ea9-42e5-af58-c9eb1014d0ab)

Reply:
No, you should delete global styles values from styles.ts ( e.x padding: 10) and apply it in a component like this: 
`style={[styles.input, globalStyles.p10]}`